tasks/automatic_updates.yml

```diff
@@ -77,3 +88,11 @@
         group: root
         mode: "0644"
         create: true
+
+    - name: Configure unattended-upgrades
+      ansible.builtin.template:
```

Comment:
`ansible.builtin.template` is destructive. Should there be a `backup: yes` for this?

Reply:
there was something with that ... iirc it was how handled the configuration if multiple files existed.
let me check.

Reply:
One thing to note with this is that any file in /etc/apt/apt.conf.d will be read as config... just using `backup: yes` is a bad idea actually

Reply:
yeah, that's the reason `backup` isn't used when it  comes to `apt` configuration (and others) in the role